zc_utils.py

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python3
+import sys; assert sys.version_info[0] >= 3, "Python 3 required."
+
```

Comment:
Weakly: this file does not generate test vectors, so it shouldn't be executable. (The assertions are run when this file is imported by others.)

Reply:
I disagree; it's useful to be able to run the assertions in this file separately.